TRANSLATIONS.md

```diff
@@ -12,6 +12,14 @@ Translated lessons should follow this naming convention:
 
 where _[language]_ is a two letter language abbreviation following the ISO 639-1 standard (e.g. `README.es.md` for Spanish and `README.nl.md` for Dutch).
 
+When translatinng lessons, link the other translations to your translated lesson by using Shields as shown 
+
+
+```` 
```

Comment:
For code sections it's three back ticks. Can you also specify that this code block is markdown